README.md

```diff
@@ -207,4 +219,10 @@ To parse an dataset, the CSV file must contain a column which is named `Model JS
     "LTLf": "F((accept_invoice) & (X[!](false)))"
   }
 ]
+```
+3. Translating the diagram to Mermaid.js.
+```yaml
```

Comment:
Doesn't it make sense to have start and end events here? Also, do we really need the label 'check invoice' twice? Can't we just use the ID?

Reply:
As in startnoneevent etc? As I'm just basing this on the parser, it deems that the 'register invoice' is a better start event than the startnoneevent.

Reply:
Added issues for this.